flags.go

```diff
@@ -106,11 +110,27 @@ func RegisterFlags() {
 	}
 }
 
+func EnvVarFlags() {
+	var value string
+
+	for f := range rtlamrFlags {
```

Comment:
Instead of looping over `rtlamrFlags`, it may be better to be more general and loop over all flags registered prior to calling this function, this will catch rtl_tcp flags as well. `flag.VisitAll` is a more appropriate way to check for environment variables matching a flag rtlamr knows about.

https://godoc.org/flag#FlagSet.VisitAll